cypress.json

```diff
@@ -13,5 +13,7 @@
       "configFile": "reporter-config.json"
     },
     "videoUploadOnPasses": false,
-    "taskTimeout": 60000
+    "taskTimeout": 60000,
+    "viewportWidth": 2000,
```

Comment:
It means we can actually see the screen in errors. Default means you can't see all results in a search for example.